pkg/api/handlers/compat/images_history.go

```diff
@@ -33,13 +33,18 @@ func HistoryImage(w http.ResponseWriter, r *http.Request) {
 	allHistory := make([]handlers.HistoryResponse, 0, len(history))
 	for _, h := range history {
 		l := handlers.HistoryResponse{
-			ID:        h.ID,
+			ID:        "sha256:" + h.ID,
```

Comment:
This line can be removed to avoid redundancy with the new if-else below.

Reply:
In my mind it was removed.